internal/httpclient/api/openapi.yaml

```diff
@@ -2893,6 +2893,7 @@ components:
       format: date-time
       title: NullTime implements sql.NullTime functionality.
       type: string
+      nullable: true
```

Comment:
You will need to do this somewhere else as this is autogenerated. In particular, it will probably require a patch along the lines of this one:

https://github.com/ory/kratos/blob/4f05d64ea8e04feb2791cf83b2296e825e3bd8ac/.schema/openapi/patches/schema.yaml#L15-L17